http/1.5/modules/ROOT/pages/http-error-status-reason-phrase-task.adoc

```diff
@@ -17,79 +17,76 @@ The example expects to receive requests in the address of http://localhost:8081/
 ** The flow calls an error handler.
 ** The HTTP Connector sets the variables in *Status Code* and *Reason* as `404 Requested user does not exist`
 
-Note that in either case, the request response doesn't display as the response body, so it won't display if you make your request through a browser window.
+Note that in either case, the request response doesn't display as the response body, so it won't display if you make your request through a browser window. To accomplish this example, you must create the Mule app, configure an HTTP global element, run, and test the app with curl commands.
```

Comment:
Again, same comment as in the other file for the 'example' text. If this is a standard thing, please ignore.